src/env_properties.h

```diff
@@ -41,7 +41,8 @@
   V(owner_symbol, "owner_symbol")                                              \
   V(onpskexchange_symbol, "onpskexchange")                                     \
   V(resource_symbol, "resource_symbol")                                        \
-  V(trigger_async_id_symbol, "trigger_async_id_symbol")
+  V(trigger_async_id_symbol, "trigger_async_id_symbol")                        \
+  V(exit_code_symbol, "exit_code_symbol")
```

Comment:
Can use a private symbol here as @joyeecheung suggested in https://github.com/nodejs/node/pull/43883#discussion_r927330749?

TLDR: v8::Privates are not visible to userland code even if you do Reflect.ownKeys() but Symbols are.

Reply:
Applied.